<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainAdminForm</class>
 <widget class="QMainWindow" name="mainAdminForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Користувачі</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTableView" name="tableView_users"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Сесії</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Додавання сесії</string>
          </property>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>140</y>
             <width>161</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Додати</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>70</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Спеціальність</string>
           </property>
          </widget>
          <widget class="QComboBox" name="DepartmentscomboBox_add">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>50</y>
             <width>361</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QComboBox" name="FacultiescomboBox_add">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>20</y>
             <width>361</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>56</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Факультет</string>
           </property>
          </widget>
          <widget class="QDateEdit" name="dateEdit_2">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>110</y>
             <width>81</width>
             <height>20</height>
            </rect>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>110</y>
             <width>33</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Кінець</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>11</x>
             <y>81</y>
             <width>43</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Початок</string>
           </property>
          </widget>
          <widget class="QDateEdit" name="dateEdit">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>80</y>
             <width>81</width>
             <height>20</height>
            </rect>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
          <zorder>label_9</zorder>
          <zorder>DepartmentscomboBox_add</zorder>
          <zorder>FacultiescomboBox_add</zorder>
          <zorder>label_10</zorder>
          <zorder>dateEdit_2</zorder>
          <zorder>label_3</zorder>
          <zorder>label_2</zorder>
          <zorder>dateEdit</zorder>
          <zorder>pushButton</zorder>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Редагування сесій</string>
          </property>
          <widget class="QComboBox" name="FacultiescomboBox">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>20</y>
             <width>311</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>56</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Факультет</string>
           </property>
          </widget>
          <widget class="QComboBox" name="DepartmentscomboBox">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>50</y>
             <width>311</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>50</y>
             <width>71</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Спеціальність</string>
           </property>
          </widget>
          <widget class="QTableView" name="tableView">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>140</y>
             <width>441</width>
             <height>192</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="facid">
           <property name="geometry">
            <rect>
             <x>420</x>
             <y>20</y>
             <width>46</width>
             <height>13</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="depid">
           <property name="geometry">
            <rect>
             <x>420</x>
             <y>50</y>
             <width>46</width>
             <height>13</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
       <zorder>groupBox_2</zorder>
       <zorder>groupBox</zorder>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Напрями підготовки</string>
       </attribute>
       <widget class="QComboBox" name="Direction_comboBox">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>20</y>
          <width>341</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="NameDirection">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>60</y>
          <width>341</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="addDirectionButton">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>100</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Додати</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Назва спеціальності</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Факультети</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Адреса</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="NameOfFacultieslineEdit"/>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QPushButton" name="AddFacultiesButton">
             <property name="text">
              <string>Додати</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Назва</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="AdressOfFaculties"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="TelephonLFaculties"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Телефон</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QTableView" name="Faculties_"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Дисципліни</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
